MaxText/max_utils.py

```diff
@@ -163,6 +163,7 @@ def create_device_mesh(config, devices=None):
   except:
     num_slices = 1
   num_devices_per_slice = num_devices//num_slices
+  max_logging.log(f"{num_devices=}, {num_slices=}, and {num_devices_per_slice=}")
```

Comment:
Probably don't do this!

Reply:
Or if you do, let me know why and possibly make a clearer summary

Reply:
ah sorry i printed the devices out to debug and i think that confused Ran. sorry my bad!

Reply:
I don't have a strong reason, but I followed the format [here]( https://github.com/google/maxtext/blob/cf100d56c548e4fed8d92557c1c86d5b2e50e640/MaxText/layers/models.py#L155). Is there any reason against it? Thanks!

Reply:
> ah sorry i printed the devices out to debug and i think that confused Ran. sorry my bad!

No worries. I thought this will be helpful. But I could just delete it if you agree.